### PR TITLE
chore/fix-dependabot-issues

### PR DESCRIPTION
**Pull request by reglim**
Fixes: #565 
Fixes: #583 
Fixes: #584 
Fixes: #575

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "@mui/material": "^5.11.4",
     "@types/jest": "^29.4.0",
     "@types/lodash": "^4.14.191",
-    "@types/node": "^18.15.0",
+    "@types/node": "^20.5.7",
```
Comment (fliiiix) [inline web/package.json:12]: we need to update the docker for that maybe?

Comment (reglim) [reply web/package.json:12]: It's already using node 20:
```FROM node:20.5 as frontend```

Comment (fliiiix) [reply web/package.json:12]: yeah just seen that :+1:

**Comment by fliiiix** [comment]
LGTM :+1:

**Pull request merged**